docs/admin/install/steps/install-python.rst

```diff
@@ -6,27 +6,32 @@ Python modules
    We're using virtualenv to install Weblate in a separate environment from your
    system. If you are not familiar with it, check virtualenv :doc:`venv:user_guide`.
 
+.. warning::
+
+   Run the following commands in an empty directory
 
 #. Create the virtualenv for Weblate:
 
    .. code-block:: sh
 
-        virtualenv --python=python3 ~/weblate-env
+        python3 -m venv .
 
 #. Activate the virtualenv for Weblate:
 
    .. code-block:: sh
 
-        . ~/weblate-env/bin/activate
+        source bin/activate
 
 #. Install Weblate including all optional dependencies:
 
    .. code-block:: sh
 
         # pkgconfig is needed to install borgbackup 1.2
-        pip install pkgconfig
+        pip3 install pkgconfig
```

Comment:
It should be `pip` as well in the virtualenv.

Reply:
Does not matter? When using Python3, can one use both pip and pip3?

Reply:
You are doing many changes which "do not matter". These are useless and make review harder - the more changes are in a PR, the easier it is to miss something important.